frontend/catalyst/api_extensions/differentiation.py

```diff
@@ -273,7 +272,7 @@ def loss(theta):
             return catalyst.value_and_grad(loss, method="auto")(theta)
 
     >>> value_and_grad_loss(1.0)
-    (array(-4.12502201), array(4.34374983))
```

Comment:
this was not the correct value, there must have been a copy-paste issue with this docstring. I verified the new values against non-qjit PL